frontend/src/components/common/ReleaseNotes/ReleaseNotes.tsx

```diff
@@ -49,6 +62,17 @@ export default function ReleaseNotes() {
             const storedAppVersion = helpers.getAppVersion();
             let releaseNotes = '';
             if (storedAppVersion && semver.lt(storedAppVersion as string, currentBuildAppVersion)) {
+              // attach a timeout which checks after 10 second of fetching release
+              // if the release request was not successful
+              setTimeout(() => {
```

Comment:
For making this extra clean, we could assign the handler to a var and remove it after the Octokit call is successful.

Reply:
done